setup/web_server_configuration.rst

```diff
@@ -95,6 +95,14 @@ directive to pass requests for PHP files to PHP FPM:
         CustomLog /var/log/apache2/project_access.log combined
     </VirtualHost>
 
+.. note::
+
+    If you run Apache via ``mod_php`` without using PHP-FPM, you can run
```

Comment:
It doesn’t really matter if PHP runs as an Apache module or via FPM if I am not mistaken.

Reply:
You are right. I've reworded this. Thanks.